From e5de12d458603a5dc2dd4febc539041025c70ebe Mon Sep 17 00:00:00 2001
From: Michael Gernoth <michael@gernoth.net>
Date: Sun, 3 Nov 2024 13:18:42 +0100
Subject: [PATCH] lsw_main: add dyld-include on apple systems

It seems oldes MacOS X versions need an additional include for
_NSGetExecutablePath.

Thanks @barracuda156

Closes #60
---
 src/dapi/src/lts/lsw_main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/dapi/src/lts/lsw_main.c b/src/dapi/src/lts/lsw_main.c
index 0e2eed3b..aa11db95 100644
--- dapi/src/lts/lsw_main.c
+++ dapi/src/lts/lsw_main.c
@@ -128,6 +128,7 @@ LTS_T   Lts_t;
 #include <limits.h>
 #include <stdint.h>
 #include <sys/types.h>
+#include <mach-o/dyld.h>
 #endif
 
 #ifdef WIN32
